interpreter/src/main/scala/pl/touk/nussknacker/engine/definition/TypeInfos.scala

```diff
@@ -75,12 +75,23 @@ object TypeInfos {
     }
   }
 
-
-  case class ClazzDefinition(clazzName: TypedClass,
+  case class ClazzDefinition(clazzName: TypingResult,
                              methods: Map[String, List[MethodInfo]],
                              staticMethods: Map[String, List[MethodInfo]]) {
+
+    def getClazz: Class[_] = this.clazzName match {
+      case TypedClass(klass, _) => klass
+      case Unknown => AnyClass
+      case _ => ???
```

Comment:
Let's replace it with throwing `IllegalAccessException` with some meaningful message that we currently support only classes and unknown in fragments parameter types